pysc2/env/sc2_env.py

```diff
@@ -282,6 +284,10 @@ def __init__(self,  # pylint: disable=invalid-name
           "Requested:\n%s\n\nActual:\n%s", game_info.options, interface)
 
     self._features = features.Features(game_info=game_info,
+                                       map_size=game_info.start_raw.map_size,
```

Comment:
map_size and camera_width can just be taken from game_info, so don't pass them in manually here.